torch/_torch_docs.py

```diff
@@ -2816,31 +2816,8 @@ def merge_dicts(*dicts):
 """.format(**common_args))
 
 add_docstr(torch.digamma, r"""
-digamma(input, *, out=None) -> Tensor
-
-Computes the logarithmic derivative of the gamma function on `input`.
-
-.. math::
-    \psi(x) = \frac{d}{dx} \ln\left(\Gamma\left(x\right)\right) = \frac{\Gamma'(x)}{\Gamma(x)}
-""" + r"""
-Args:
-    input (Tensor): the tensor to compute the digamma function on
-
-Keyword args:
-    {out}
-
-.. note::  This function is similar to SciPy's `scipy.special.digamma`.
-
-.. note::  From PyTorch 1.8 onwards, the digamma function returns `-Inf` for `0`.
-           Previously it returned `NaN` for `0`.
-
-Example::
-
-    >>> a = torch.tensor([1, 0.5])
-    >>> torch.digamma(a)
-    tensor([-0.5772, -1.9635])
-""".format(**common_args))
-
+Alias for :func:`torch.special.psi`.
```

Comment:
Shouldn't torch.digamma and torch.special.psi both be aliases for torch.special.digamma? 

https://docs.scipy.org/doc/scipy/reference/generated/scipy.special.digamma.html